nameservice/Makefile

```diff
@@ -15,12 +15,12 @@ include Makefile.ledger
 all: install
 
 install: go.sum
-		go install -mod=readonly $(BUILD_FLAGS) ./cmd/nsd
+		go install  $(BUILD_FLAGS) ./cmd/nsd
```

Comment:
what is the reason to remove -mod=readonly? i do believe this updates some deps and this is not wanted.

Reply:
Was needed for mine to build. Can’t remember why now.

Reply:
can you undo this change, otherwise you will be pulling new versions which are not supported.

Reply:
Done and pushed.

Reply:
😄 thanks